builtin/logical/pki/path_tidy.go

```diff
@@ -872,10 +898,31 @@ func (b *backend) doTidyCertStore(ctx context.Context, req *logical.Request, log
 
 		cert, err := x509.ParseCertificate(certEntry.Value)
 		if err != nil {
-			return fmt.Errorf("unable to parse stored certificate with serial %q: %w", serial, err)
+
+			// only log warning once
+			if !haveWarned {
```

Comment:
Should the same logic be added to `doTidyRevocationStore` as well I wonder?

I _think_ so, so perhaps we should wait until #677 lands to rebase on top of that (so that we correctly handle tidying each individually)....

Though I don't think we should add the `config.InvalidCerts` as an or to `if config.RevokedCerts || config.IssuerAssocs {` as we're already doing one iteration over the certs in this pass. That means we might miss certs (if for some reason, we have a revocation entry but no actual cert), but that should be fine.

My 2c!

Reply:
I think that would be a good idea, maybe in the future we can have the option for `- verbose` in the case that a user wants to debug with details for whatever reason.

Reply:
@cipherboy I will wait for #677 to land before I rebase this PR and modify the logging style in `doTidyRevocationStore`, then after this PR lands, I will open up the draft PR for using pagination in tidy and rebase there :D